docker/r10k/Dockerfile

```diff
@@ -52,7 +55,9 @@ SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 RUN chmod a+x /adduser.sh /docker-entrypoint.sh && \
 # Add a puppet user to run r10k as for consistency with puppetserver
     /adduser.sh && \
-    chown -R puppet: /docker-entrypoint.d /docker-entrypoint.sh && \
+    chown -R "${PP_USER}:" /docker-entrypoint.d /docker-entrypoint.sh && \
+    chmod -R +x            /docker-entrypoint.d /docker-entrypoint.sh && \
+    mkdir -p /docker-custom-entrypoint.d && \
```

Comment:
No, because the folder was meant to be a mount-point and is therefore empty, so I don't really see the reason for that.
The general idea was to allow people to mount a folder on the host with scripts there would be executed in the r10k container